### PR TITLE
Removing rc1 qualifier

### PR DESCRIPTION
**Pull request by sgup432**
Signed-off-by: Sagar Upadhyaya <sagar.upadhyaya.121@gmail.com>

**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
https://github.com/opensearch-project/performance-analyzer/issues/210

**Describe the solution you are proposing**

**Describe alternatives you've considered**

**Additional context**

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by sruti1312** [comment]
For this PR we need to also update the ```plugin-descriptor.properties``` file

https://github.com/opensearch-project/performance-analyzer/blob/81d019af51a62d42592905cffc51f9c77cb8aaac/src/main/resources/plugin-descriptor.properties#L30

File: build.gradle [hunk 1]
```diff
@@ -6,11 +6,11 @@
 buildscript {
 
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "2.0.0-SNAPSHOT")
```
Comment (sruti1312) [inline build.gradle:9]: ```[plugin-descriptor.properties]``` also needs to be updated.

File: build.gradle [hunk 2]
```diff
@@ -6,11 +6,11 @@
 buildscript {
 
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
```
Comment (sruti1312) [inline build.gradle:9]: [plugin-descriptor.properties](https://github.com/opensearch-project/performance-analyzer/blob/81d019af51a62d42592905cffc51f9c77cb8aaac/src/main/resources/plugin-descriptor.properties) also needs to be updated.

**Review by sruti1312 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (sgup432) [reply build.gradle:9]: Thanks for pointing it out. Added.

**Review by sruti1312 — APPROVED** [review]

**Review by dzane17 — APPROVED** [review]

**Pull request merged**